doc/modules/experimental.rst

```diff
@@ -0,0 +1,41 @@
+.. _experimental_ref:
+
+:mod:`nilearn.experimental`: Experimental Modules
+=================================================
+
+.. automodule:: nilearn.experimental
+   :no-members:
+   :no-inherited-members:
+
+:mod:`nilearn.experimental.surface`
+-----------------------------------
+
+.. automodule:: nilearn.experimental.surface
```

Comment:
Won't this lead to having some entries twice?

Reply:
removing this, removed the warnings for me.

Reply:
Yea that's not supposed to be there 🤦

Reply:
Let's call that a "is Rémi paying attention test". 🤗